src/main/scala/Configs.scala

```diff
@@ -153,6 +153,7 @@ class DefaultConfig extends Config (
       }
       case BuildRoCC => Nil
       case RoccNMemChannels => site(BuildRoCC).map(_.nMemChannels).foldLeft(0)(_ + _)
+      case RoccNPTWPorts => site(BuildRoCC).map(_.nPTWPorts).foldLeft(0)(_ + _)
```

Comment:
There should be another commit which changes all Rocc references to RoCC, so I'm submitting this PR again only with the updated rocket hash tag.